core/jobs_test.py

```diff
@@ -702,6 +817,19 @@ def _handle_incoming_event(
             cls, active_realtime_layer, event_type, exp_id, unused_exp_version,
             unused_state_name, unused_session_id, unused_params,
             unused_play_type):
+        """Records incoming events in the given realtime layer.
+
+        Args:
+            active_realtime_layer: int. The currently active realtime datastore
+                layer.
+            event_type: str. The event triggered by a student.
```

Comment:
Our convention is learners rather than students.

```suggestion
            event_type: str. The type of event triggered by a learner.
```